client/executor/wasmi/src/lib.rs

```diff
@@ -306,16 +325,19 @@ impl<'a> wasmi::Externals for FunctionExecutor<'a> {
 		-> Result<Option<wasmi::RuntimeValue>, wasmi::Trap>
 	{
 		let mut args = args.as_ref().iter().copied().map(Into::into);
-		let function = self.host_functions.get(index).ok_or_else(||
-			Error::from(
-				format!("Could not find host function with index: {}", index),
-			)
-		)?;
-
-		function.execute(self, &mut args)
-			.map_err(|msg| Error::FunctionExecution(function.name().to_string(), msg))
-			.map_err(wasmi::Trap::from)
-			.map(|v| v.map(Into::into))
+
+		if let Some(function) = self.host_functions.get(index) {
+			function.execute(self, &mut args)
+				.map_err(|msg| Error::FunctionExecution(function.name().to_string(), msg))
+				.map_err(wasmi::Trap::from)
+				.map(|v| v.map(Into::into))
+		} else if self.enable_stub {
+			Err(Error::from(
+				format!("functions do not exist: {}", self.missing_functions.join(", "))
```

Comment:
Please only print the actual function that was tried to be called (by using the unique index generated in `resolve`).

Reply:
If I don't print all the not existing functions, why am I gathering them in a Vec? This implementation here was printing the function that was tried: https://github.com/paritytech/substrate/pull/4550/commits/e19948e8efa91f2ef745db609967bc7726285311

Reply:
Because you only want to know which function exactly was called. It does not helps us to know that 50 functions are not provided ^^

Reply:
Ok so after discussion it finally :bulb: in my mind. It's the name of the missing export we need, not the name of the function in the module. Will fix this

Reply:
done